woodwork/table_accessor.py

```diff
@@ -580,6 +586,7 @@ def wrapper(*args, **kwargs):
                     else:
                         copied_schema = self._schema._get_subset_schema(list(self._dataframe.columns))
                         result.ww.init(schema=copied_schema)
+                        result.ww.make_index = self.make_index
```

Comment:
Another slightly odd situation here. 

Since I don't want to pass `make_index=True` to the `init` above this, I figure it's better to set the value after since I might expect a user to want that value propagated to the new DataFrame.

Reply:
Makes sense to me.